### PR TITLE
Test the output of `SortitionPool.withdrawRewards`

### PR DESCRIPTION
**Pull request by beaushinkle**
Follow up to #163, we alter `test/sortitionPoolTest.js` to make assertions about the return values of `withdrawRewards`.

Unfortunately since `withdrawRewards` isn't a `view` function we have to extract the return data from the events or logs.

Tagging @pdyraga for review

### CODE REVIEW

File: test/sortitionPoolTest.js [hunk 1]
```diff
@@ -2,6 +2,19 @@ const chai = require("chai")
...
+  owner,
+  operatorAddress,
+  beneficiaryAddress,
+) {
+  const tx = await pool
+    .connect(owner)
+    .withdrawRewards(operatorAddress, beneficiaryAddress)
+  const withdrawLogs = await tx.wait()
+  return parseInt(withdrawLogs.logs[0].data)
+}
```
Comment (pdyraga) [inline test/sortitionPoolTest.js:16]: We can also do it this way:
```
    async function withdrawRewards(
      pool,
      owner,
      operatorAddress,
      beneficiaryAddress,
    ) {
      const amount = await pool
        .connect(owner)
        .callStatic.withdrawRewards(operatorAddress, beneficiaryAddress)
      await pool.connect(owner).withdrawRewards(operatorAddress, beneficiaryAddress)
      return amount
    }
```

File: test/sortitionPoolTest.js [hunk 2]
```diff
@@ -261,7 +274,7 @@ describe("SortitionPool", () => {
   describe("pool rewards", async () => {
```
Comment (pdyraga) [inline test/sortitionPoolTest.js:274]: What if we define `withdrawRewards()` function only in this context? It an auxiliary function to test pool rewards and is not used anywhere else.

```
describe("pool rewards", async () => {
    async function withdrawRewards(
      pool,
      owner,
      operatorAddress,
      beneficiaryAddress,
    ) {
      // (...)
    }
// (...)
```

Comment (pdyraga) [inline test/sortitionPoolTest.js:274]: What if we define `withdrawRewards()` function only in this context? It an auxiliary function to test pool rewards and is not used anywhere else.

```
describe("pool rewards", async () => {
    async function withdrawRewards(
      pool,
      owner,
      operatorAddress,
      beneficiaryAddress,
    ) {
      // (...)
    }
// (...)
```

Comment (beaushinkle) [reply test/sortitionPoolTest.js:261]: this looks like a duplicate comment!

File: test/sortitionPoolTest.js (hunk 1 above)
Comment (beaushinkle) [reply test/sortitionPoolTest.js]: [b8fd7c8](https://github.com/keep-network/sortition-pools/pull/166/commits/b8fd7c821b3af00ea7e608950f9475f658e6dcc6)

**Comment by pdyraga** [comment]
@beaushinkle Linter is complaining. Once it's fixed I am good to merge.

**Pull request merged**